### PR TITLE
Update event syndication enablement content

### PR DESCRIPTION
**Pull request by rwellon**
Edits to existing content for enabling event syndication.

### CODE REVIEW

**Comment by rwellon** [comment]
@miq-bot assign @agrare @nasark
@miq-bot add_reviewer @Fryguy

**Comment by rwellon** [comment]
@miq-bot assign @nasark

**Comment by rwellon** [comment]
@miq-bot add-label quinteros/yes?

**Review by nasark — APPROVED** [review]
Looks great, thanks!

**Review by agrare — APPROVED** [review]

File: general_configuration/_topics/configuration.md [hunk 1]
```diff
@@ -3083,15 +3083,28 @@ to access the server, and restart the {{ site.data.product.title_short }} applia
...
-    :keep_events: 6.months
-    :purge_window_size: 1000
-  :syndicate_events: false
-```
-3. Set the `:syndicate_events:` value to `true`
+You can enable events syndication in the `Advanced` settings:
+
+1. From the menu, go to **Settings** > **Application Settings**.
+
+2. Expand the **Settings** accordian. When you select the **Settings** accordion, you are automatically taken to the server list under **Zones**.
```
Comment (Fryguy) [inline general_configuration/_topics/configuration.md:3090]: Minor typo here

```suggestion
2. Expand the **Settings** accordion. When you select the **Settings** accordion, you are automatically taken to the server list under **Zones**.
```

**Comment by rwellon** [comment]
I'll fix this in another PR

File: general_configuration/_topics/configuration.md (hunk 1 above)
Comment (rwellon) [reply general_configuration/_topics/configuration.md:3090]: I'll fix this in another PR

**Comment by Fryguy** [comment]
Backported to `quinteros` in commit a154162ce6dc9df7282c42709ab15a8a7aa953bd.

```text
commit a154162ce6dc9df7282c42709ab15a8a7aa953bd
Author: Adam Grare <adam@grare.com>
Date:   Mon Oct 16 09:12:00 2023 -0400

    Merge pull request #1751 from rwellon/RW-EventSyndicationEdits
    
    Update event syndication enablement content
    
    (cherry picked from commit ace9e5a9becb805fb4528603e76c4abef0792124)
```

**Pull request merged**